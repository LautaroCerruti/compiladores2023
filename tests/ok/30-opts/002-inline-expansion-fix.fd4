let rec triang (w x y z:Nat) : Nat = 
    ifz x then (ifz x then y else triang w x y (z-2)) else (fun (l : Nat) -> ifz y then print "finished " y else triang w x (y-1) z) 1

let a : Nat = print "finished2 " (triang 1 2 1000 3)