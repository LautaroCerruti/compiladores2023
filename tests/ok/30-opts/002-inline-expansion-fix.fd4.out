finished 0
finished2 0
